partials/twitter.php

```diff
@@ -1,12 +1,29 @@
-<?php 
-  $twitter_handle = IGV_get_option( '_igv_twitter_handle' ); 
+<?php
+// Get twitter feed
+$twitter_feed = get_twitter_feed( IGV_get_option( '_igv_twitter_handle' ) );
```

Comment:
tbh i feel the get option would be better done inside the function after checked if that transient exists, surely that saves db queries?